static/app/views/settings/organizationDeveloperSettings/index.tsx

```diff
@@ -180,8 +205,19 @@ class OrganizationDeveloperSettings extends AsyncView<Props, State> {
             }
           )}
         </Alert>
-        {this.renderExternalIntegrations()}
-        {this.renderInternalIntegrations()}
+
+        <NavTabs underlined>
+          {tabs.map(tabTuple => (
```

Comment:
Instead of calling the variable `tabTuple`, since those don't really exist in JS, I'd destructure the array wholesale:

```js
tabs.map( ([tabType, tabLabel]) => {...} )
```
This way you don't have to jump around to figure out what `tabTuple[0]` or `tabTuple[1]` are representing.